YAXLibTests/SerializerOptionsTests.cs

```diff
@@ -1,75 +0,0 @@
-﻿// Copyright (C) Sina Iravanian, Julian Verdurmen, axuno gGmbH and other contributors.
-// Licensed under the MIT license.
-
-using System.Xml.Linq;
-using FluentAssertions;
-using NUnit.Framework;
-using YAXLib;
-using YAXLib.Enums;
-using YAXLib.Options;
-
-namespace YAXLibTests
-{
-    [TestFixture]
-    public class SerializerOptionsTests
```

Comment:
why are these removed?

Reply:
Guess not useful.

Reply:
Removed SerializerOptionsTests: containing no tests, but still covered 100% because in use ~1 million times